longest-string/longest-string.js

```diff
@@ -0,0 +1,36 @@
+class longestString {
```

Comment:
since you're using es6 syntax here, i suggest using es6 syntax when defining variables within the class. you should use `const` or `let` rather than `var`